README.md

```diff
@@ -312,6 +312,14 @@ In such case, the `reason` attribute of the annotation should point to correspon
 If the `@OpenShiftTest` annotation is configured to use the strategy `UsingQuarkusPluginDeploymentStrategy` and the module contains the `quarkus-openshift` Quarkus extension, the test framework will use Quarkus to build and deploy the module into OpenShift.
 Note that the property `quarkus.kubernetes.deploy` must NOT be set in the module configuration in order to avoid to perform the deployment at build time.  
 
+#### Deployment using Quarkus docker build strategy
+
+If the `@OpenShiftTest` annotation is configured to use the strategy `UsingQuarkusPluginDeploymentStrategy.UsingQuarkusPluginDeploymentStrategy`
```

Comment:
@mjurc Probably typo in `UsingQuarkusPluginDeploymentStrategy. UsingDockerStrategy`